test/addon_runners/chrome/main.js

```diff
@@ -84,7 +90,13 @@ exports.testChromeInPanel = function(assert, done) {
     });
     panel.port.emit('echo');
   });
-  panel.show();
+
+  // NOTE: hack needed to prevents random test timeout (probably due to the
+  // setTimeout used in the 'panel.show' method:
+  // https://github.com/mozilla/addon-sdk/blob/a19bd8c/lib/sdk/panel/utils.js#L78)
+  intervalId = setInterval( _ => {
```

Comment:
@kumar303 this test case actually runs inside the Firefox instance launched by the jpm's mocha test case and it is not reused for running the other test addons in the jpm test suite.

In any case it is absolutely reasonable to cleanup the setInterval after the test timeouts.

I'm going to add it in a follow-ups.